source/mainnet/net/guides/run-node-macos.rst

```diff
@@ -263,10 +263,51 @@ When you start a node for the first time, it can take a while to synchronize the
 node with the rest of the network, since it has to get all blocks from its
 peers.
 
-You can improve the performance by downloading the blocks before starting the
-node. While it will still take time to process the blocks, it will typically be
+You can improve the startup time by downloading the blocks from an out-of-band catchup service
+before starting the node. While it will still take time to process the blocks, it will typically be
 faster than requesting them from peers.
 
+.. note::
+
+   Catchup data for mainnet does not work with a testnet node and vice versa.  Make sure to use the
+   correct URL to the block file index for your node.
+
+#. Specify the URL to the block file index in the service file:
+
+   - For mainnet:
+
+     - Edit ``/Library/Concordium Node/LaunchDaemons/software.concordium.mainnet.node.plist`` as an
+       administrator and add the following in the *EnviromentVariables* section::
+
+       <key>CONCORDIUM_NODE_CONSENSUS_DOWNLOAD_BLOCKS_FROM</key>
+       <string>https://catchup.mainnet.concordium.software/blocks.idx</string>
+
+   - For testnet:
+
+     - Edit ``/Library/Concordium Node/LaunchDaemons/software.concordium.testnet.node.plist`` as an
+       administrator and add the following in the *EnviromentVariables* section::
+
+       <key>CONCORDIUM_NODE_CONSENSUS_DOWNLOAD_BLOCKS_FROM</key>
+       <string>https://catchup.testnet.concordium.com/blocks.idx</string>
+
+
+#. Restart the appropriate node by running the application **Concordium Node Stop [Mainnet/Testnet]** (if running) and then
+   **Concordium Node Start [Mainnet/Testnet]**.
+
+#. Go to the mainnet or testnet dashboard to monitor when the node has caught up with its
+   peers on the blockchain. You do so by comparing the finalized length of the
+   chain with the length of your node. If they match, your node has caught up.
+
+.. note::
+
+   It is recommended to keep the `CONCORDIUM_NODE_CONSENSUS_DOWNLOAD_BLOCKS_FROM` variable set in the node configuration, unlike `CONCORDIUM_NODE_CONSENSUS_IMPORT_BLOCKS_FROM`.
+   The former permits incremental out-of-band catchup starting from the best block already present in the node database.
+   The latter does not and slows down the node startup significantly.
```

Comment:
This is not a recommendation we have agreed on and it should not be stated as such. Once the node is caught up they should remove this from the configuration file. For normal operation this is useless (e.g., if they did a node upgrade or maintenance), will slow down node startup a bit, and will put a load on our servers.